pyproject.toml

```diff
@@ -22,7 +22,7 @@ exclude = '''
   | build/
   | dist/
   | ddtrace/(
-    (?!(compat|monkey|span|__init__|_hooks)\.py$)[^/]+\.py$
+    (?!(compat|monkey|span|__init__|_hooks|provider)\.py$)[^/]+\.py$
```

Comment:
?

Reply:
is it a new file? why wasn't it having issues before?

Reply:
This rule ignores all Python files in the `ddtrace/` directory except for the ones explicitly listed there. `ddtrace/provider.py` was formerly ignored but it got formatted and so now we shouldn't ignore it 😛

Reply:
It's a 2 line change.. I don't think it's that big a deal. If you feel strongly then I'll revert it

Reply:
It's always a big deal to me, because if you have to revert things, then you revert more thing that you should.
PRs are free. No reason to pack everything.